packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js

```diff
@@ -70,6 +70,22 @@ export default /* @ngInject */ ($stateProvider) => {
 
       operation: /* @ngInject */ (PciPrivateNetworks, projectId) =>
         PciPrivateNetworks.getVrackCreationOperation(projectId),
+      gateways: /* @ngInject */ (
+        $q,
+        PciPrivateNetworks,
+        privateNetworksRegions,
+        projectId,
+      ) => {
+        return $q
+          .all(
+            privateNetworksRegions.map((region) =>
+              PciPrivateNetworks.getGateways(projectId, region)
+                .then((data) => data)
+                .catch(() => []),
+            ),
+          )
+          .then((gatewaysData) => gatewaysData.flat());
+      },
```

Comment:
**nitpick:** Use the regionalised API `/cloud/project/{serviceName}/aggregated/gateway` and replce multiple calls with the single one.

Reply:
Taken care.